presto-docs/src/main/sphinx/release/release-0.149.rst

```diff
@@ -4,10 +4,22 @@ Release 0.149
 
 General Changes
 ---------------
+* Fix :func:`trim` and :func:`rtrim` function to produce more intuitive result
```

Comment:
If a string ends with a valid whitespace UTF8 code point followed by an invalid UTF8 sequence, the code before fix can corrupt the valid code point.

This may or may not be considered a bug in Presto because Presto didn't explicitly what kind of garbage output is allowed when input is garbage.

In airlift.slice, it was specified in trim and rtrim that invalid sequence won't be trimmed. Therefore, it was a bug there.